api-reference/beta/api/applicationsecurityprofile_update.md

```diff
@@ -0,0 +1,143 @@
+# Update applicationSecurityProfile
+
+Update the user-editable properties of a applicationSecurityProfile object. **This method must include the applicationSecurityProfileId as a parameter.**
+
+## Permissions
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) |   SecurityEvents.ReadWrite.All  |
+|Delegated (personal Microsoft account) |  Not supported  |
+|Application |  SecurityEvents.ReadWrite.All |
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+This method must include the applicationSecurityProfileId as a parameter.
+
+```http
+PATCH /security/applicationSecurityProfiles/{applicationSecurityProfile_Id}
+```
+
+## Request headers
+
+| Name       | Description|
+|:-----------|:-----------|
+| Authorization  | Bearer {code} (required)|
+|Prefer | return=representation |
+
+## Request body
+
+In the request body, supply a JSON representation of the values for the values for relevant fields that should be updated. Only the fields that can be updated for an applicationSecurityProfile are listed below. Existing properties that are not included in the request body will maintain their previous values. For best performance you shouldn't include existing values that haven't changed.
+
+| Property   | Type |Description|
+|:---------------|:--------|:----------|
+|tags|String|User-definable labels that can be applied to an applicationSecurityProfile and can serve as filter conditions (e.g. "HVA", "SAW", etc.)|
+

```

Comment:
Don't use "e.g." or "etc." as they are difficult to localize. Instead, use "for example".